utils/ccdatofhir/mapping_resource_allergy_intolerance.bal

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// --------------------------------------------------------------------------------------------#
+// Source C-CDA to FHIR - Resource Mappings
+// --------------------------------------------------------------------------------------------#
+
+import ballerinax/health.fhir.r4;
+
+# map CCDA Allergy to FHIR AllergyIntolerance
```

Comment:
```suggestion
# Map CCDA Allergy to FHIR AllergyIntolerance.
```

Refer: https://learn-ballerina.github.io/best_practices/documenting_the_source_code.html

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/bd15548be31a7e76b4962973508cdedd6f8fd58a

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/bfbad6e342e0df400c6fffecc9682cf0f99e19b7